tgui/packages/tgui/styles/colors.scss

```diff
@@ -29,6 +29,8 @@ $average: #f08f11 !default;
 $bad: #db2828 !default;
 $label: #7e90a7 !default;
 
+$gold: #f2a918 !default;
```

Comment:
I am fine with adding this to base colors. Though, if we want to start a custom color group, can you add a new line with a comment for that? Also while we are here, can we add another comment line to mark State colors?

Reply:
Good idea